RealmSwift/SwiftUI.swift

```diff
@@ -265,6 +265,55 @@ private class ObservableStorage<ObservedType>: ObservableObject where ObservedTy
     }
 }
 
+@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.0, *)
+private class ObservableResultsStorage<T>: ObservableStorage<T> where T: RealmSubscribable & ThreadConfined & Equatable {
+    var setupHasRun = false
+    func didSet() {
+        if setupHasRun {
+            setupValue()
+            setupHasRun = true
+        }
+    }
+
+    func setupValue() {
```

Comment:
I would probably add a message or comment so we know this is been override by the storage on each property wrappers